dvdmaker, moviemk, wmp*: add page (#22085)
tldr-pages/tldr: pages/windows/dvdmaker.md

# dvdmaker

> > Windows DVD Maker.

Reviewer: ```suggestion
> Create and burn video DVDs.
```

> Note: This program has been deprecated.
> More information: <https://learn.microsoft.com/previous-versions/windows/desktop/wmmdvdm/command-line-options-for-windows-dvd-maker>.

- Launch Windows DVD Maker:

`dvdmaker`

- Specify the DVD disk burner drive:

`dvdmaker -drive:{{F}}`

- Import assets predefined by a Windows DVD Maker import file list XML file:

`dvdmaker {{path\to\file.xml}}`

> - Reset the Windows DVD Maker configuration into factory settings (Warning: This will reset your app settings):

Reviewer: ```suggestion
- Reset Windows DVD Maker configuration to factory settings (Warning: This will reset your app settings):
```
